text/2000-const-generics.md

```diff
@@ -0,0 +1,192 @@
+- Feature Name: `const_generics`
+- Start Date: 2017-05-01
+- RFC PR: [rust-lang/rfcs#2000](https://github.com/rust-lang/rfcs/pull/2000)
+- Rust Issue: [rust-lang/rust#44580](https://github.com/rust-lang/rust/issues/44580)
+- Translators: [[@yanganto](https://github.com/yanganto)]
+- Commit: [The commit link this page based on](https://github.com/rust-tw/rfcs-tw/blob/725bf172f383a21cf74189feb5073705778d2206/text/2000-const-generics.md)
+- Updated: 2021-03-26
+
+# 總結
+[總結]: #總結
+
+允許所有類型（type）以作為常數值（constant values）的形式成為泛型（generic）；由此將允許使用者以撰寫所有陣列類型（array types）之抽象的實作（impls）。
+
+# 動機
+[動機]: #動機
+
+現今Rust具有一種以常數參數化（parametric）類型︰內建陣列類形`[T; LEN]`。然而，常數泛型（const generics）並非一級特徵，使用者無法以常數形式定義自己的泛型，且不能對所有的陣列實作特徵（traits）。
```

Comment:
```suggestion
現今 Rust 具有一種以常數參數化（parametric）類型︰內建陣列類形`[T; LEN]`。然而，常數泛型（const generics）並非一級特徵，使用者無法以常數形式定義自己的泛型，且不能對所有的陣列實作特徵（traits）。
```
LGTM! Just need to add spaces between english characters and chinese characters.